rotkehlchen/tests/unit/decoders/test_superchain_bridge.py

```diff
@@ -592,13 +583,11 @@ def test_withdraw_eth_base_to_ethereum_bridge(base_inquirer, base_accounts):
 @pytest.mark.parametrize('base_accounts', [['0x779e4b47c3Dea5689233821dFcf429E0485eF116']])
 def test_withdraw_erc20_base_to_ethereum_bridge(base_inquirer, base_accounts):
     evmhash = deserialize_evm_tx_hash('0xbe4e54e77cb700f2755b236d7823295129c4d5e22fdc87df8058274bc0fefab1')  # noqa: E501
-    events, _ = get_decoded_events_of_transaction(evm_inquirer=base_inquirer, tx_hash=evmhash)
-    gas_amount, deposit_amount = '0.000029210771559657', '49352.072702'
-    user_address = base_accounts[0]
+    (events, _), gas_amount, deposit_amount, user_address, timestamp = get_decoded_events_of_transaction(evm_inquirer=base_inquirer, tx_hash=evmhash), '0.000029210771559657', '49352.072702', base_accounts[0], TimestampMS(1729293119000)  # noqa: E501
```

Comment:
why did you touch them? It's good to move the timestamp but I would leave `events, _ = get_decoded_events_of_transaction(evm_inquirer=base_inquirer, tx_hash=evmhash)` in its own line. If you don't need to modify this file revert the changes since it adds noise to the commit

Reply:
they're nitpicks from a previous pr. lefteris said to add them in a follow up pr.